ses-resources/cephadm

```diff
@@ -0,0 +1,200 @@
+#!/bin/bash
+
+# Source this to run cephadm-specific validations
+# for the SUSE Enterprise Storage supportutils plugin
+
+
+#############################################################
+rpm_list="$SES_SES_RESOURCES_DIR"/rpm-list
+for thisrpm in $(cat "$rpm_list"); do
+    validate_rpm_if_installed "$thisrpm"
+done
+
+#############################################################
+section_header "Ceph cluster status"
+
+mkdir $LOG/ceph
```

Comment:
This line should be removed (it's already handled on line 26 of the main ses script, and including it here results in an error message, as the directory already exists)